src/store/houses/mutations.ts

```diff
@@ -8,4 +8,22 @@ export const mutations: MutationTree<HousesStateInterface> & Mutations = {
   [HousesMutationsEnum.SET_MODAL_DISPLAY]: (state, payload) => {
     state.displayModal = payload;
   },
+  [HousesMutationsEnum.EDIT_HOUSE]: (state, payload) => {
+    state.houses?.forEach((el: HouseInterface) => {
+      if (el.id === payload.id) {
+        el.quantity_flat = payload.quantity_flat;
+        el.house_area = payload.house_area;
+        el.adjoining_area = payload.adjoining_area;
+        el.address = payload.address;
+      }
+    });
```

Comment:
You can extract this logic to the method in the model and then just use this method here

Reply:
May i suggest to refactor this part on next sprint? I still nedd a little bit more time to figure out how models work in TS, so i can do it properly

Reply:
Yep, okay, but remind me please about this item.